botnim/vector_store/vector_store_es.py

```diff
@@ -258,28 +260,11 @@ def delete_existing_files(self, context_, vector_store, file_names):
             return 0
 
     def update_tools(self, context_, vector_store):
-        # vector_store is now just the index name string
-        if len(self.tools) == 0:
-            self.tools.append({
-                "type": "function",
-                "function": {
-                    "name": f"search_{vector_store}",
-                    "description": f"Semantic search the '{vector_store}' vector store",
-                    "parameters": {
-                        "type": "object",
-                        "properties": {
-                            "query": {
-                                "type": "string",
-                                "description": "The query string to use for searching"
-                            }
-                        },
-                        "required": ["query"]
-                    }
-                }
-            })
+        # No need to create tools here as they're handled in sync.py
```

Comment:
Can you explain the logic for this change?

Reply:
Tools are part of the Assistant configuration, not the vector store implementation. Vector stores should focus on managing embeddings and search functionality.

Reply:
another reason for externalizing it into a tools folder was to be able to define several different search tools, to serve different needs.

Reply:
based on our talked, I moved the tools creation process back here